dusty/commands/test.py

```diff
@@ -67,56 +67,51 @@ def pull_repos_and_sync(app_or_lib_name, pull_repos=False):
     spec = expanded_specs.get_app_or_lib(app_or_lib_name)
     nfs.update_nfs_with_repos(get_same_container_repos_from_spec(spec))
 
-def run_app_or_lib_tests(app_or_lib_name, suite_name, test_arguments, should_exit=True, force_recreate=False):
-    expanded_specs = get_expanded_libs_specs()
-    spec = expanded_specs.get_app_or_lib(app_or_lib_name)
-    test_command = _construct_test_command(spec, suite_name, test_arguments)
-    try:
-        ensure_test_image(app_or_lib_name, expanded_specs, force_recreate=force_recreate)
-    except ImageCreationError as e:
-        log_to_client('Failed to create test container with error {}'.format(e.code))
-        sys.exit(e.code)
-    exit_code = _run_tests_with_image(expanded_specs, app_or_lib_name, test_command, suite_name)
-    if should_exit:
-        log_to_client('TESTS {} {}'.format(suite_name, 'FAILED' if exit_code != 0 else 'PASSED'))
-        sys.exit(exit_code)
-    return exit_code
+def ensure_image(app_or_lib_name, force_recreate):
```

Comment:
Maybe rename this to something that indicates it will also update even if an existing image is there?